EncoreTickets.SDK/Inventory/IInventoryServiceApi.cs

```diff
@@ -65,6 +86,7 @@ public interface IInventoryServiceApi : IServiceApi
         /// <param name="quantity">quantity of seats needed</param>
         /// <param name="performance">performance: if nothing is sent, current time will be used</param>
         /// <returns>Array of availability seat</returns>
+        [Obsolete]
```

Comment:
This endpoint is still used by Europa, why is this method marked obsolete? Also why do you not use parameters for this attribute?

Reply:
As BE guys answered, this endpoint should not exist in v4 inventory and was added to the 4th version of the API by confusion. They recommended not using it and using an endpoint that ends with /area, this is the GetAggregateSeatAvailability method

Reply:
Ok. Still, this should be mentioned in the corresponding attribute parameter. Also, could you please clarify what should be used by Europa to get the fields that are not present in the new endpoint?